app/Repositories/PhotoRepository.php

```diff
@@ -39,18 +40,22 @@ public function __construct(AWS $aws, Registrar $registrar)
      * @param  int $signupId
      * @return \Rogue\Models\Photo|null
      */
-    public function create(array $data, $signupId)
+    public function create(array $data, Signup $signup)
```

Comment:
I think that maybe adding the `northstar_id` into the `photo` array in the `SignupsController`  before creating the post might be the nicest way to do it. That way the migration requests can still be a little nicer but the `/posts` endpoint is still happy